aquadoggo_cli/config.toml

```diff
@@ -0,0 +1,167 @@
+# aquadoggo configuration file
+#
+# Copy this file to the location where aquadoggo will be run or refer to it
+# using the -c command line argument when running aquadoggo. Replace the
+# example values with your own desired configuration.
+#
+# NOTE: Paths in this file follow the XDG Base Directory Specification for
+# Linux. You might want to adjust these values for your regarding operating
+# system.
+
+# ~~~~~~~
+# Schemas
+# ~~~~~~~
+
+# List of schema ids which a node will replicate and expose on the GraphQL API.
+#
+# When allowing a schema you automatically opt into announcing, replicating and
+# materializing documents connected to it, supporting applications which are
+# dependent on this data.
+#
+# It is recommended to set this list to all schema ids your own application
+# should support. Defaults to an empty list.
+#
+# WARNING: When set empty, your node will support _all_ schema ids. This is
+# useful for experimentation and local development but _not_ recommended for
+# production settings.
+#
+supported_schema_ids = [
```

Comment:
It would be good to be able to configure a node which doesn't support any schema, this way you could run a relay which aids in discovery & connectivity but _doesn't_ itself replicate any data. Ideally by setting these two configuration args as so:

```
supported_schema_ids = []
im_a_relay = true
```

Could this be achieved if we allow a wildcard value in the schema ids list `[*]`?

Reply:
Or make it so that when no `supported_schema_ids` value is provided (it's commented out in the config) then we assume this to mean _any_ are supported.

Reply:
I'll think about the wildcard thingy, that would be the most explicit.

We can't write `[*]` because this is not valid toml but `"*"` would work.

Reply:
> This is the current behaviour 👍

Ah, ok, I misread that from the code then, thought the default of an empty set was to support everything.

Reply:
As long as there are ways to account for the three situations:

- support no schema
- dynamically support all schema
- support a set of specific schema

Then I think that's all we need, doesn't have to be with the wildcard approach, just omitting the support schema config is totally fine.

Reply:
> Then I think that's all we need, doesn't have to be with the wildcard approach, just omitting the support schema config is totally fine.

Yes, this we already support. The wildcard would be an extra to make it more explicit.